Gems/Atom/RPI/Code/Source/RPI.Reflect/Image/Image.cpp

```diff
@@ -67,6 +70,19 @@ namespace AZ
         RHI::ResultCode Image::UpdateImageContents(const RHI::ImageUpdateRequest& request)
         {
             RHI::ImagePool* imagePool = azrtti_cast<RHI::ImagePool*> (m_image->GetPool());
+
+            // aefimov font hack to update a streaming image
```

Comment:
FFont uses a streaming image created from a texture data, see
m_fontStreamingImage = RPI::StreamingImage::CreateFromCpuData
But UpdateImageContents expects an attachment image (I believe), anyway it must be something with RHI::ImagePool, not RHI::StreamingImagePool. So the original code produces nullptr for imagePool (see teh conversion above).
The nasty hack below uses attachment image pool to update the image. I have to override the image's pool to bypass internal check. I see no quick and nice solution, please give e piece of advice.